recipe/meta.yaml

```
@@ -12,16 +12,18 @@ source:
 
 build:
   number: 0
```

is it the first version or shall we update the build number?

```suggestion
  number: 1
```

---

This is the first version. There is no yet amply in defaults.